docs/conf.py

```diff
@@ -41,16 +41,16 @@
 
 # General information about the project.
 project = u'WSGI-Kerberos'
-copyright = u'2013, Michael Komitee'
+copyright = u'2020, D. E. Shaw & Co., L.P.'
```

Comment:
IIUC, the copyright notice should include all years and copyright holders, not just the latest. You can abbreviate consecutive years with a hyphen like 2013-2019.

Reply:
Also, IIUC you can add “and contributors” to cover the individuals who’ve contributed.

Reply:
Looks like the LICENSE file needs the same treatment:
https://github.com/deshaw/wsgi-kerberos/blob/714ba41ff2b62e7d0c976eeff197839742044946/LICENSE#L1

Reply:
Might even be worth pasting the license (with the fixed copyright notice) in a comment header in wsgi_kerberos.py and in test_wsgi_kerberos.py (several licenses recommend doing this since individual files in a project often get separated from the other files and reproduced without the LICENSE file getting included).

Reply:
This is the standard license text for all D.E.Shaw projects. I'd leave it as is for now, but FYI @mlucool if you think we should change it.

Reply:
Fixed the years. From what I understand the whole "and contributors" is debatable in the community. It's a fact they do license the work and not grant us copyright ownership, but the lack of saying that does negate their claims. I'd prefer to leave it as is.

Reply:
A little more guidance from google: https://opensource.google/docs/releasing/authors/